ref-exchange/src/token_receiver.rs

```diff
@@ -34,51 +36,64 @@ impl Contract {
     ) -> Vec<(AccountId, Balance)> {
```

Comment:
I know you're going to reimplement this method. Maybe just comment it out for now?

Reply:
removed those codes